models/validation-inference-scripts/phishing-models/phish-bert-inference-script.py

```diff
@@ -21,6 +21,9 @@
     --output phishing-email-validation-output.jsonlines
 """
 
+import cudf
+from cudf.core.subword_tokenizer import SubwordTokenizer
+
```

Comment:
You should put a comment before and after these imports indicating why they need to go first and to prevent isort from rearranging them.